modules/casper.js

```diff
@@ -254,11 +287,47 @@ Casper.prototype.back = function back() {
  * @param  String  url     The url to download
  * @param  String  method  The method to use, optional: default GET
  * @param  String  data    The data to send, optional
+ * @param  Boolean async   Asynchroneous request? (default: false)
  * @return string          Base64 encoded result
  */
-Casper.prototype.base64encode = function base64encode(url, method, data) {
+Casper.prototype.base64encode = function base64encode(url, method, data, async) {
     "use strict";
-    return this.callUtils("getBase64", url, method, data);
+    async = ((typeof(async) !== "undefined")? async : false);
+	if ((phantom.version.major > 1) && (!async)) {
```

Comment:
I'd like to directly set async to true if phantomjs is > 2 instead of throw an error.